src/applications/mhv-landing-page/components/MedicalRecords.jsx

```diff
@@ -30,98 +31,106 @@ export const recordTypes = [
 
 const toKebabCase = (phrase = '') => phrase.toLowerCase().replace(/\s+/g, '-');
 
-const MedicalRecords = ({ blueButtonUrl }) => (
-  <div
-    className={classnames(
-      'vads-l-grid-container',
-      'large-screen:vads-u-padding-x--0',
-      'vads-u-margin-bottom--5',
-    )}
-    data-testid="mhvMedicalRecords"
-  >
-    <VaBreadcrumbs breadcrumbList={breadcrumbList} />
+export const pageTitle = 'Medical records';
 
-    <div className="vads-u-display--flex vads-u-flex-wrap--wrap">
-      <h1 className="vads-u-margin-bottom--0">Medical records</h1>
+const MedicalRecords = ({ blueButtonUrl }) => {
+  useEffect(() => {
+    document.title = titleCase(`${pageTitle} | Veterans Affairs`);
+  }, []);
+
+  return (
```

Comment:
Note: These following changes are indentation changes except for the use of `pageTitle` in line 53.